crypto/balance.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Finobo
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crypto
+
+type Balance interface {
+	Balance() string
```

Comment:
Balance shouldn't be part of the crypto package. It tempting cus you use a key to get the balance. But you'd need either go to the protocol client to query the balance or ensure that you have done the query before creating a key. This would bloat the package and create external dependencies. Looking at the rest of the PR, it doesn't seem like its called anywhere.